pebble_tool/commands/base.py

```diff
@@ -110,107 +101,186 @@ def __call__(self, args):
 
     def _connect(self, args):
         self._set_debugging(args.v)
-        if getattr(args, 'phone', None):
-            return self._connect_phone(args.phone)
-        elif getattr(args, 'qemu', None):
-            return self._connect_qemu(args.qemu)
-        elif getattr(args, 'emulator', None):
-            return self._connect_emulator(args.emulator, args.sdk)
-        elif getattr(args, 'cloudpebble', None):
-            return self._connect_cloudpebble()
-        elif getattr(args, 'serial', None):
-            return self._connect_serial(args.serial)
-        else:
-            if 'phone' in self.valid_connections and 'PEBBLE_PHONE' in os.environ:
-                return self._connect_phone(os.environ['PEBBLE_PHONE'])
-            elif 'qemu' in self.valid_connections and 'PEBBLE_QEMU' in os.environ:
-                return self._connect_qemu(os.environ['PEBBLE_QEMU'])
-            elif 'cloudpebble' in self.valid_connections and os.environ.get('PEBBLE_CLOUDPEBBLE', False):
-                return self._connect_cloudpebble()
-            elif 'serial' in self.valid_connections and 'PEBBLE_BT_SERIAL' in os.environ:
-                return self._connect_serial(os.environ['PEBBLE_BT_SERIAL'])
-            elif 'emulator' in self.valid_connections:
-                running = []
-                emulator_platform = None
-                emulator_sdk = None
-                if 'PEBBLE_EMULATOR' in os.environ:
-                    emulator_platform = os.environ['PEBBLE_EMULATOR']
-                    if emulator_platform not in pebble_platforms:
-                        raise ToolError("PEBBLE_EMULATOR is set to '{}', which is not a valid platform "
-                                        "(pick from {})".format(emulator_platform, ', '.join(pebble_platforms)))
-                    emulator_sdk = os.environ.get('PEBBLE_EMULATOR_VERSION', sdk_version())
-                else:
```

Comment:
You've removed the functionality implemented by this block: if exactly one emulator is running, and no connection is specified, we assume you meant that one.